bot/cogs/help_channels.py

```diff
@@ -217,23 +221,18 @@ def create_name_queue(self) -> deque:
         return channel
 
     @staticmethod
-    def get_position(channel: discord.TextChannel, destination: discord.CategoryChannel) -> int:
-        """Return the position to sort the `channel` at the bottom if moved to `destination`."""
-        log.trace(f"Getting bottom position for #{channel} ({channel.id}).")
-
-        if not destination.channels:
-            # If the destination category is empty, use the first position
-            position = 1
-        else:
-            # Else use the maximum position int + 1
-            position = max(c.position for c in destination.channels) + 1
-
-        log.trace(
-            f"Position of #{channel} ({channel.id}) in {destination.name} will be {position} "
-            f"(was {channel.position})."
-        )
+    def get_clean_channel_name(channel: discord.TextChannel) -> str:
+        """Return a clean channel name without status emojis prefix."""
+        try:
+            # Try to remove the status prefix using the index of "help-"
+            name = channel.name[channel.name.index("help-"):]
```

Comment:
The prefix is configurable so "help-" shouldn't be hard coded.

Reply:
Good catch.